src/web/mjs/connectors/SmackJeeves.mjs

```diff
@@ -1,130 +1,99 @@
 import Connector from '../engine/Connector.mjs';
 
-/**
- *
- */
 export default class SmackJeeves extends Connector {
 
-    /**
-     *
-     */
     constructor() {
         super();
         super.id = 'smackjeeves';
         super.label = 'Smack Jeeves';
         this.tags = [ 'webtoon', 'english' ];
-        this.url = 'http://www.smackjeeves.com';
+        this.url = 'https://www.smackjeeves.com';
+        this.he = require('he');
     }
 
-    /**
-     *
-     */
-    _getMangaListFromPages( mangaPageLinks, index ) {
-        index = index || 0;
-        return this.fetchDOM( mangaPageLinks[ index ], 'div.comic-container a.card', 5 )
-            .then( data => {
-                let mangaList = data.map( element => {
-                    return {
-                        id: this.getRelativeLink( element ),
-                        title: element.querySelector( 'div.title' ).textContent.trim()
-                    };
-                } );
-                if( index < mangaPageLinks.length - 1 ) {
-                    return this._getMangaListFromPages( mangaPageLinks, index + 1 )
-                        .then( mangas => mangas.concat( mangaList ) );
-                } else {
-                    return Promise.resolve( mangaList );
+    async _getCategories( url) {
+        const script = `
+            new Promise((resolve, reject) => {
+                try {
+                    resolve(cmnData);
+                } catch(error) {
+                    reject(error);
                 }
-            } );
+            });
+        `;
+        const request = new Request(new URL(url), this.requestOptions);
+        const data = await Engine.Request.fetchUI(request, script);
+
+        return data.navigation.items;
```

Comment:
May use `data.navigation.items.map(item => item.val)`, since the val is the only thing that is used later on

Reply:
I like that. `.map()` is still a thing where i learn something everytime i use it.